assignment-2/strava-god/src/app/main-navbar/main-navbar.component.html

```diff
@@ -0,0 +1,38 @@
+<mat-sidenav-container class="sidenav-container">
+
+    <mat-sidenav #drawer class="sidenav" fixedInViewport="false" 
+        [ngClass]="{hidden: !(isHandset$ | async)}"
```

Comment:
isHandset should be local variabel in component. Subscribe in OnInit and unsubscribe i OnDestroy - set local variabel in subscribe callback